server/game/cards/11-TCaR/RosenbaumsGolem.js

```diff
@@ -0,0 +1,52 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class RosenbaumsGolem extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Rosenbaum\'s Golem',
+            cost: ability.costs.bootSelf(),
+            playType: ['shootout'],
+            target: {
+                activePromptTitle: 'Choose an opposing dude',
+                cardCondition: { location: 'play area', controller: 'opponent' },
+                cardType: ['dude'],
+                gameAction: 'boot'
+            },
+            message: context => this.game.addMessage('{0} uses {1} to boot {2}', context.player, this, context.target),
+            handler: context => {
+                this.game.resolveGameAction(GameActions.bootCard({ card: context.target }), context).thenExecute(() => {
+                    context.game.promptForYesNo(context.player, {
+                        title: 'Do you want to give ' + context.target.title + ' +5 value?',
+                        onYes: () => {
+                            this.game.addMessage('{0} uses {1} to give {2} +5 value', context.player, this, context.target);
```

Comment:
The +5 value effect is missing.

Reply:
oh, doh! i'll fix it over lunch today.

Reply:
😄

Reply:
added the missing effect :heavy_check_mark: